docs/maintenance/VERSIONING.md

```diff
@@ -28,9 +27,22 @@ During these periods, we manually publish `canary` releases until we are happy w
 
 ## Dependant Versions
 
+> See [Versioning > Dependant Version Upgrades](./versioning/dependant-version-upgrades.mdx) for maintenance steps to update these versions.
+
+### ESLint
+
+> The version range of ESLint currently supported is `^6.0.0 || ^7.0.0 || ^8.0.0`.
```

Comment:
Not introduced in this PR, but I strongly suggest using admonitions like `:::info` instead. Quotes should be used for quotes and not other things.

Reply:
Heh, an old habit from working in raw Markdown. +1 on the _semantics_ of switching away from quotes. But I do prefer the _visual_ of a quote because admonitions take the extra line to indicate their type.

<img width="869" alt="Screenshot of a Note admonition" src="https://user-images.githubusercontent.com/3335181/208319098-c325fec4-7ab4-4846-a940-24e699cd05fe.png">

Has there been any discussion about making single-line admonitions? My GitHub+Google searching is failing me, but I can make a Discussion if not.

Reply:
No discussion yet, but IMO it's fine to make the doc page a bit less densely populated :)

Reply:
https://github.com/facebook/docusaurus/discussions/8458

Reply:
@Josh-Cena what do you think - should we stick with the larger admonitions, or smaller-but-semantically-off quotes? I'll defer to your suggestion 😄